### DISCUSSION TITLE
Important tip or recommendation for RustyBox

### DISCUSSION THREAD

**Discussion by VincentAraneo** [SuperTux/supertux#2911] [General]
I don't know if I can post it here, but it's necessary for RustyBox for the future of supertux

Hey Rusty, I think you already know the case that happened with your backup from 6 months ago, and in the end you made it private or deleted it. So I want to give you an important piece of advice so that it never happens again. Your assets or levels in your account, you have to delete them and make a private repository that you put your assets there. Basically you have to delete Rusty-Box/SuperTux Branches that are the next thing, (unless you already have a Pull Request), if that branche doesn't have any Pull Request or you haven't done it yet, you have to delete that branche that doesn't have a Pull Request!, for example, I don't know if you remember, but you made a Branche that literally, has all Icy Island Remastered and that you published it 3 months ago, and also another branche that had the new yeti,  If you don't believe me, here's the evidence
![importanttip_image1](https://github.com/SuperTux/supertux/assets/139823928/d8a13625-cf95-4e5d-bb7b-01a1f2c36822)
![importanttip_image2](https://github.com/SuperTux/supertux/assets/139823928/a2d33c4d-c5da-43c7-8bca-7b53e1e6688c)

You see?, I have to delete those branches, you see, I have those files when I find out, and I don't. The Icy Island Remastered Archive I won't do anything to it at all, I just downloaded it to play it, not to post it on youtube or put it into my mod, because if I don't I get in trouble or my mod will lose meaning. I recommend that you delete those Branches, as it doesn't have a Pull Request, only the ones that already have a Pull Request, will not be deleted. I recommend you put the branches that don't have any Pull Request to a private repository that you need to create to be more organized, That repository has to be private so that no one sees it, and when it's ready, there you can do that branche again and make a pull request, so it will be impossible for anyone and me to see it. It's a tip so that all your assets are protected and that no one can download it.

If you've made it this far from the text, I just hope the advice helps, so you can protect your things. I know this text has nothing to do with supertux and only has to do with the future of supertux and its protection, but this tip is important for you rustybox, so you can protect your assets at all costs. If you have a question, comment here or on Discord, so I can help you on these issues. I hope the advice helps.

**Comment by tylerandari13** [comment]
Im dying of laughter right now.

**Reply by VincentAraneo** [reply]
I know that several will not care about it and will only take it off-topic. But this will help Rustybox so that its assets are not seen by just anyone

**Comment by CrystalDaEevee** [comment]
lmao what.

**Comment by Rusty-Box** [comment]
Your lack of brain capacity is astonishing. Guess what, those branches exist because soon I will have to make those into PRs anyway. Did I put anything else on my fork that is not meant to be seen? No! There is nothing "off" about them being there as I'm preparing all these files to be made into PRs in the next coming months. Who are you to tell me how to do my github activity, huh?! Grow up. It's not my problem you can't get your shit together and stop digging through all of my account like a desperate baby looking for milk to digest.

**Reply by CrystalDaEevee** [reply]
w

**Reply by VincentAraneo** [reply]
Well, if your assets or complements start leaking, don't start blaming me, it's just your problem and you have to fix it if it happens. Just be careful if someone badder than me ends up publishing World 1 Remastered (This is not to offend, it's just to warn you what can happen)

**Reply by tylerandari13** [reply]
> Well, if your assets or complements start leaking, don't start blaming me, it's just your problem and you have to fix it if it happens. Just be careful if someone badder than me ends up publishing World 1 Remastered (This is not to offend, it's just to warn you what can happen)

Theyre not leaks. Theyre publicly avaliable, and if someone says theyre leaks theyre not only lying but theyre also an idiot.

**Reply by retcinder** [reply]
"Leaking", my brother in christ, its being publicly developed (as its a copyleft software project) so its not "leaking". Nobody will do what you're suggesting could happen because its makes literally no sense to do.

**Reply by CrystalDaEevee** [reply]
My brother in christ the game is open source

**Reply by retcinder** [reply]
Also, while I have this open, please both:
- Educate yourself on what Open Source is along with licenses (since this is seemingly the second time you've misunderstood this)
- Come back when you comply with the GitHub Terms Of Service (specifically, [Chapter A, Point 7 ("The User, "You" and "Your")](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#a-definitions), or, for a more in-depth section of the GitHub Terms of Service, [Chapter B, Section 3 (Account Requirements)](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#3-account-requirements))

**Reply by tylerandari13** [reply]
> "Leaking", my brother in christ, its being publicly developed (as its a copyleft software project) so its not "leaking". Nobody will do what you're suggesting could happen because its makes literally no sense to do.

I remember seeing a SuperTux addon called Industrial Island. Someone tried to "leak" it for some reason even though its open sourced and publicly avaliable. This is probably one of the biggest acts of stupidity ive seen in a while, especially when the team behind the addon had to talk for a really long time with the dude to get him to take down the stuff they didnt want up. Please dont be like this guy. Dont package publicly avaliable stuff and call it a leak, and especially dont be stubborn about taking it down. I know you can do better than this.

**Reply by VincentAraneo** [reply]
Now I understand, they didn't leak it. So for example if rustybox publishes world 3 on github, it doesn't mean it's a leak, now I understood well that industrial island v2 was not a leaked build.

**Comment by komaros1** [comment]
What

**Reply by retcinder** [reply]
Don't ask...

**Comment by Rusty-Box** [comment]
> Well, if your assets or complements start leaking, don't start blaming me, it's just your problem and you have to fix it if it happens. Just be careful if someone badder than me ends up publishing World 1 Remastered (This is not to offend, it's just to warn you what can happen)

The remaster for world 1 has been in private testing, via playtesters, for a while now and is soon going into public testing, hence I've already upload initial version of them to my fork. This is not the same as posting images that where not yet meant for public eyes thus do they not fall under the "open source" umbrella to begin with as I have not given them that license!

**Reply by VincentAraneo** [reply]
Ahh now understanding better. I thought it was just future Pull Requests that haven't been released yet and are just there anymore. And now that I understand, I want to ask you something. Why did you put a public backup for everyone without knowing that someone can download it just like me?

**Comment by MatusGuy** [comment]
your j's are fake
